apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -293,7 +293,7 @@ void AprilTagDetector::imageCb(const sensor_msgs::PointCloud2ConstPtr& cloud,
       bool validPose = true;
 
       // The maximum allowed angle delta for each axis
-      if ((diffRoll > plane_angle_threshold_) || (diffPitch > plane_angle_threshold_))
+      if (number_of_frames_to_capture_ == 1 && (diffRoll > plane_angle_threshold_) || (diffPitch > plane_angle_threshold_))
```

Comment:
is there a reason we went with turning it this off with the number of frames to capture instead of adding a field? this feels like it could be confusing and tricky in the future.

Reply:
This is a quick and dirty implementation. the correct one is just what you suggest :-)

Reply:
Konrad, please add comments explaining the hack so we can fix more easily later.